### PR TITLE
"tx not found" and "block not found" error pages

### PR DESCRIPTION
**Pull request by maxgrapps**
1) Add _errors.scss to css/compontents folder
2) Connect _errors.scss at the bottom of app.scss
3) Add "errors-img" folder and "errors-tpl" folder to static/images

NOTE: This pages made for ethereum mainnet theme. Only difference for another themes is in images. I'll upload in in another pull requests.

screencapture-localhost-4000-tx-0x32e2e16e6a06a1e67cd023a8a68c54a5d590f9426a6fb672041410dabad31e96-2019-05-21-19_19_09

screencapture-localhost-4000-blocks-24-transactions-2019-05-21-19_18_50

### CODE REVIEW

File: apps/block_scout_web/priv/gettext/default.pot [hunk 1]
```diff
@@ -1749,6 +1706,13 @@ msgid "of"
 msgstr ""
 
 #, elixir-format
+<<<<<<< HEAD
```
Comment (vbaranov) [inline apps/block_scout_web/priv/gettext/default.pot]: @maxgrapps please remove manual merging artefacts

File: apps/block_scout_web/priv/gettext/default.pot [hunk 2]
```diff
@@ -1776,4 +1740,8 @@ msgstr ""
 #, elixir-format
 #: lib/block_scout_web/templates/address_logs/index.html.eex:77
 msgid "There are no logs for this address."
+<<<<<<< HEAD
```
Comment (vbaranov) [inline apps/block_scout_web/priv/gettext/default.pot]: @maxgrapps please remove manual merging artefacts

**Comment by maxgrapps** [comment]
@vbaranov completed

**Comment by maxgrapps** [comment]
@vbaranov Thanks. We have unique pictures on error pages for each theme. How we can set it? And Should I create 10 separated pull requests for each theme? Or we can add some dynamic code to eex files?

**Comment by vbaranov** [comment]
@maxgrapps we will not do this dynamic. We will update each branch before each deployment with the correct image. No need to create 10 PRs.

**Comment by maxgrapps** [comment]
@vbaranov Okay. do you need something from my side? (All images already uploaded)

**Comment by vbaranov** [comment]
@maxgrapps for this task no

**Pull request merged**